Ricochet/Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -241,6 +367,60 @@ private IEnumerator RespawnPowerUpRoutine(GameObject powerUp)
         powerUp.SetActive(true);
     }
 
+    private void MatchTimer()
+    {
+        if (currentMatchTime > 0)
+        {
+            currentMatchTime -= Time.deltaTime;
+            string minutes = ((int)(currentMatchTime / 60)).ToString();
+            int seconds_num = (int)(currentMatchTime % 60);
+            string seconds;
+            if (seconds_num < 10)
+            {
+                seconds = '0' + seconds_num.ToString();
+            }
+            else
+            {
+                seconds = seconds_num.ToString();
+            }
+            gameTimerText.text = minutes + ':' + seconds;
+        }
+        else
+        {
+            //if we want to disable the players during the win screen not sure exactly what to do here so need some team input
```

Comment:
Should probably move to a seperate screen with no players at all

Reply:
I figured this would become Kyle's stat screen once he has that done but for now should I just comment this out or leave it.

Reply:
Just leave it for now I guess